package.json

```diff
@@ -48,8 +48,8 @@
     "prettier-check": "prettier --check .",
     "prepare": "husky install",
     "pack": "cd utils && node -r ts-eager/register ./pack.ts",
-    "version:prepare": "changeset version && pnpm install --no-frozen-lockfile",
-    "release": "changeset publish"
+    "ci:version": "changeset version && pnpm install --no-frozen-lockfile",
+    "ci:publish": "pnpm publish -r && changeset tag"
```

Comment:
Will this push the tags too? The docs are not clear.
https://github.com/changesets/changesets/blob/main/docs/command-line-options.md#tag

Reply:
`changeset/action` pushes the tags after invoking this script.